platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/AccountsManagerTest.java

```diff
@@ -15,10 +23,37 @@ public void testIsExpired() {
     long futureValue = now + 1;
     long immediatePast = now - 1;
 
-    Assert.assertTrue(AccountsManager.isExpired(now, defaultValue));
-    Assert.assertTrue(AccountsManager.isExpired(now, tooOld));
+    assertTrue(AccountsManager.isExpired(now, defaultValue));
+    assertTrue(AccountsManager.isExpired(now, tooOld));
+
+    assertFalse(AccountsManager.isExpired(now, futureValue));
+    assertFalse(AccountsManager.isExpired(now, immediatePast));
+  }
+
+  @Test
+  public void checksSkuTokenExternalManagement() {
+    SharedPreferences mockedSharedPreferences = mock(SharedPreferences.class);
+    when(mockedSharedPreferences.getString(KEY_PREFERENCE_SKU_TOKEN, "")).thenReturn("external-sku-token");
+    boolean isNotManaged = false;
+    AccountsManager theAccountsManager = new AccountsManager(mockedSharedPreferences, isNotManaged);
+
+    String skuToken = theAccountsManager.getSkuToken();
+
+    assertEquals("external-sku-token", skuToken);
+  }
+
+  @Test
+  public void checksSkuTokenInternalManagement() {
+    SharedPreferences mockedSharedPreferences = mock(SharedPreferences.class, RETURNS_DEEP_STUBS);
```

Comment:
TIL about the `Mockito#RETURNS_DEEP_STUBS`, but I don't think it's necessary here?

Reply:
It is necessary for https://github.com/mapbox/mapbox-gl-native/blob/434eacd60cd8bdf933894c6bfe9b4fe32ad560c5/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/AccountsManager.java#L99-L101 and https://github.com/mapbox/mapbox-gl-native/blob/434eacd60cd8bdf933894c6bfe9b4fe32ad560c5/platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/AccountsManager.java#L141-L144 We could create the stubs (and get rid off `RETURNS_DEEP_STUBS`) but would be overkilling especially because we're not using them at all in the tests.